hivemind/moe/server/connection_handler.py

```diff
@@ -6,11 +6,12 @@
 import grpc
 import torch
 
-from hivemind.proto import runtime_pb2, runtime_pb2_grpc as runtime_grpc
 from hivemind.moe.server.expert_backend import ExpertBackend
-from hivemind.utils import get_logger, Endpoint, nested_flatten
+from hivemind.proto import runtime_pb2
+from hivemind.proto import runtime_pb2_grpc as runtime_grpc
```

Comment:
Maybe [combine_as_imports](https://pycqa.github.io/isort/docs/configuration/options.html#combine-as-imports) might be helpful in this case